README.md

```diff
@@ -118,6 +118,23 @@ sentry_set_commits(
 )
 ```
 
+#### Create deploy
+
+Creates a new release deployment for a project on Sentry.
+
+```ruby
+sentry_create_deploy(
+  version: '...',
+  app_identifier: '...', # pass in the bundle_identifer of your app
```

Comment:
`m`: I'm not sure if we should remove the `app_identifier`. It's anyway just prepended to the version so users can do that directly in the version. I think it could be even a bit confusing having it.

Reply:
Added this because it's the same approach in other actions. Should remove it just here?

Reply:
agree with @denrase , every action has `app_identifier` so I'd keep it for the sake of compatibility, and if people are using the other commands, they know what to place in here anyway